electroncash/contacts.py

```diff
@@ -382,3 +386,28 @@ def find(self, *, address: str = None, name: str = None, type: str = None,
             if type is not None and c.type != type:
                 continue
             yield c
+
+    def update_lns_contacts(self, window) -> int:
```

Comment:
This function assumes that the LNS subsystem is running in the context of the Qt app. Perhaps you weren't aware of this but: This assumption doesn't hold for the Android app, the iOS app, or the text-based "gui" version of electron cash.  The rule is that anything in the `electroncash/` subfolder should not touch Qt or know about Qt or assume Qt exists.

I will fix this class of error in the commits I am preparing (this is not the only place this happens).